<!doctype html>
<html class="no-js" lang="en-US">

<head>
    <meta charset="utf-8">
    <title>iDEC 2022 | SUSTech_Shenzhen</title>
    <link rel="icon" href="static/picture/logo.ico" type="image/x-icon">
    <meta name="description" content="">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <link rel="stylesheet" href="static/css/animate.css">
    <link rel="stylesheet" href="static/css/slinky.min.css">
    <link rel="stylesheet" href="static/css/themify-icons.css">
    <link rel="stylesheet" href="static/css/fontawesome.min.css">
    <link rel="stylesheet" href="static/css/bootstrap.min.css">
    <link rel="stylesheet" href="static/css/swiper.min.css">
    <link rel="stylesheet" href="static/css/normalize.css">
    <link rel="stylesheet" href="static/css/main.css">
    <link rel="stylesheet" href="static/css/reset.css">
    <link rel="stylesheet" href="static/css/saastrace-search.css">
    <link rel="stylesheet" href="static/css/saastrace.css">
    <link rel="stylesheet" href="static/css/responsive.css">

    <meta name="theme-color" content="#fafafa">
</head>

<body>
<!--[if IE]>
<p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
<![endif]-->



<div class="loader-wrapper">
        <span class="loader">
            <span class="loader-inner"></span>
        </span><!-- .loader -->
</div><!-- .loader-wrapper -->


<!-- Header area -->

<header class="header-area saastrace-initial saastrace-primary-header saastrace-white-bg saastrace-initial-header">
    <div class="container-fluid">
        <div class="row saastrace-shrink no-gutters">
            <div class="col">
                <nav class="navbar navbar-expand-md navbar-light saastrace-navbar saastrace-initial-navbar">

                    <a class="navbar-brand single-service" href="index.html">
                        <img class="dark-logo" src="static/picture/logosmall.png" alt="SaasTrace" style="width: 60%">
                    </a>
                    <div class="collapse navbar-collapse" id="navbarSupportedContent">
                        <ul class="navbar-nav ml-auto">
                            <li class="menu-item menu-item-has-children"><a class="active" href="#">Wetlab</a>
                                <ul class="sub-menu">
                                    <li><a href="Wetlab_Design.html" >Design</a></li>
                                    <li><a href="Wetlab_Engineering.html" >Engineering</a></li>
                                    <li><a href="Wetlab_Notebook.html" >Notebook</a></li>
                                </ul>
                            </li>
                            <li class="menu-item menu-item-has-children"><a href="#">Drylab</a>
                                <ul class="sub-menu">
                                    <li><a href="Drylab_Modeling_Overview.html">Overview</a></li>
                                    <li><a href="Drylab_Modeling_Evolution.html">Directed Evolution</a></li>
                                </ul>
                            </li>
                            <li class="menu-item menu-item-has-children"><a href="#">Parts</a>
                                <ul class="sub-menu">
                                    <li><a href="Parts_Parts-list.html">Parts list</a></li>
                                    <li><a href="Parts_Improvement.html">Improvement</a></li>
                                </ul>
                            </li>
                            <li class="menu-item " ><a class="active" href="HP_Education.html">Education</a>
                            </li>
                            <li class="menu-item menu-item-has-children"><a href="#">Entrepreneurship</a>
                                <ul class="sub-menu">
                                    <li><a href="Entrepreneurship_VCL.html">EPIC VCL workshops</a></li>
                                    <li><a href="Entrepreneurship_Summer.html">Summer bootcamp</a></li>
                                    <li><a href="Entrepreneurship_Demo.html">EPIC demo day</a></li>
                                </ul>
                            </li>
                            <li class="menu-item menu-item-has-children"><a href="#">Team</a>
                                <ul class="sub-menu">
                                    <li><a href="Team_Members.html">Members</a></li>
                                    <li><a href="Team_Attribution.html">Attribution</a></li>
                                    <li><a href="Team_Gallery.html">Gallery</a></li>
                                    <li><a href="Team_Collaboration.html">Collaboration</a></li>
                                    <li><a href="Team_Partnership.html">Partnership</a></li>
                                </ul>
                            </li>
                        </ul>


                    </div><!-- .collapse .navbar-collapse -->
                </nav>
            </div><!-- .col-xs-12 -->
        </div><!-- .row -->
    </div><!-- .container -->
</header><!-- .header-area -->


<!-- ================================================================= -->
<!-- ======================== Mobile Menu Area ======================= -->
<!-- ================================================================= -->

<div class="saastrace-mobile-menu-area">
    <div class="saastrace-mobile-menu">
        <div class="container-fluid">
            <div class="row no-gutters">
                <div class="col saastrace-shrink">
                    <ul class="active">
                        <li>
                            <a class="mobile-logo" href="index.html"><img src="static/picture/logo.png" alt="SaasTrace"></a>
                        </li>
                        <li>
                            <a href="#">
                                    <span>
                                        <span class="bar"></span>
                                        <span class="bar"></span>
                                        <span class="bar"></span>
                                    </span>
                            </a>
                            <ul>
                                <li><a href="#" class="next">Wetlab</a>
                                    <ul>
                                        <li><a href="Wetlab_Design.html">Design</a></li>
                                        <li><a href="Wetlab_Engineering.html">Engineering</a></li>
                                        <li><a href="Wetlab_Experiments.html">Experiments</a></li>
                                        <li><a href="Wetlab_Results.html">Results</a></li>
                                        <li><a href="Wetlab_Safety.html">Safety</a></li>
                                        <li><a href="Wetlab_Notebook.html">Notebook</a></li>
                                        <li><a href="Wetlab_Protocal.html">Protocal</a></li>
                                    </ul>
                                </li>
                                <li><a href="#">Drylab</a>
                                    <ul>
                                        <li><a href="Drylab_Modeling_Overview.html">Overview</a></li>
                                        <li><a href="Drylab_Modeling_Evolution.html">Directed Evolution</a></li>
                                        <li><a href="Drylab_Modeling_Tyrian.html">Tyrian Purple</a></li>
                                        <li><a href="Drylab_Modeling_Reference.html">Reference</a></li>
                                    </ul>
                                </li>
                                <li><a href="#">Parts</a>
                                    <ul>
                                        <li><a href="Parts_Parts-list.html">Parts list</a></li>
                                        <li><a href="Parts_Improvement.html">Improvement</a></li>
                                    </ul>
                                </li>
                                <li><a href="#">HP</a>
                                    <ul>
                                        <li><a href="HP_Integrated-HP.html">Integrated HP</a></li>
                                        <li><a href="HP_Education.html">Education</a></li>
                                    </ul>
                                </li>
                                <li><a href="#">Entrepreneurship</a>
                                    <ul>
                                        <li><a href="Entrepreneurship_VCL.html">EPIC VCL workshops</a></li>
                                        <li><a href="Entrepreneurship_Summer.html">Summer bootcamp</a></li>
                                        <li><a href="Entrepreneurship_Demo.html">EPIC demo day</a></li>
                                    </ul>
                                </li>
                                <li><a href="#">Team</a>
                                    <ul>
                                        <li><a href="Team_Members.html">Members</a></li>
                                        <li><a href="Team_Attribution.html">Attribution</a></li>
                                        <li><a href="Team_Gallery.html">Team Events</a></li>
                                        <li><a href="Team_Collaboration.html">Collaboration</a></li>
                                        <li><a href="Team_Partnership.html">Partnership</a></li>
                                    </ul>
                                </li>
                            </ul>
                        </li>
                    </ul>
                </div><!-- .col -->
            </div><!-- .row -->
        </div><!-- .container-fluid -->
    </div><!-- .saastrace-mobile-menu -->
</div><!-- .saastrace-mobile-menu -->



<div class="image-container">
    <img src="static/picture/blog-3.jpg" alt="About Image">
</div><!-- .image-container -->

<section class="saastrace-our-gallery-area" style="padding-bottom: 15px">
    <div class="bubble bubble-one"></div>
    <div class="bubble bubble-two"></div>
    <div class="bubble bubble-three"></div>
    <div class="container-fluid">
        <div class="row">
            <div class="col">
                <div class="our-gallery-content text-center">
                    <h2 class="saastrace-title wow fadeInUp" data-wow-delay="200ms">Engineering</h2>
                </div><!-- .our-gallery-content -->
            </div><!-- .col -->
        </div><!-- .row -->
    </div><!-- .container -->
</section><!-- .saastrace-our-gallery-area -->


<!-- Blog -->
<div class="saastrace-blog-area">
    <div class="container">
        <div class="row">
            <div class="col-lg-9">
                <div class="saastrace-blog">
                    <article class="saastrace-single-blog-post">
                        <div class="single-news-post-wrapper">
                            <div class="single-news-post-content">
                                <h2><a href="blog-single.html">Overview</a></h2>
                                <div class="blog-post-news-description">
                                    <p>Engineer and optimize TPE project is our core task for wetlab. Just like the strategy of directed evolution, we believe that an interesting project needs to be “evoluted”. We followed a “Design → Build → Test → Learn” loop to achieve the “evolution” of TPE project. There are two parts of optimization in the project: </p><br>
                                    <p>We use directed evolution to develop the process of synthesis from two-cell system to one-cell system, with an expected optimization controlling system of endogenous TnaA enzyme in E.coli. </p><br>
                                    <p>We introduce a reversible glucose protection group to make the production of tyrian purple dye more stable and convenient for industrial scale-up.
                                    </p><br>
                                    <p>Thus, we did the “Design → Build → Test → Learn” loop for each of optimization, making them become highlights in TPE project.</p><br>
                                </div><!-- .blog-post-news-description -->
                            </div><!-- .single-news-post-content -->
                        </div><!-- .single-news-post-wrapper -->
                    </article><!-- .saastrace-single-blog-post -->
                    <article class="saastrace-single-blog-post">
                        <div class="single-news-post-wrapper">
                            <div class="single-news-post-content">
                                <h2><a>Establishing a pathway from Trp to 6-bromoindole</a></h2>
                                <h3><a>Design</a></h3>
                                <div class="blog-post-news-description">
                                    <p>SttH is a trp-6-halogenase that requires FADH2 as a co-factor to convert tryptophan (Trp) into 6-Br-trp. Tryptophanase TnaA can cleave Trp into indole or cleave 6-Br-Trp into 6-Br-indole. Thus, 6-Br-indole can be synthesized via two routes using SttH and TnaA: Previous conversion of Trp to indole followed by bromination of indole or bromination of Trp followed by conversion to 6-Br-indole.
                                    </p><br>
                                    <img src="static/picture/engineering1.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 1 </a></h5>
                                        <span class="news-date">Two possible pathways from Trp to 6-bromoindole</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <h3><a>Build</a></h3>
                                    <p>To validate the two pathways, all the enzymatic steps using TnaA or SttH were individually evaluated through whole-cell reactions. To prevent the effect of endogenous TnaA expression, a ΔtnaA strain of E. coli was used and TnaA was overexpressed using a plasmid when needed.</p><br>
                                    <p>Thanks to 2021 iGEM team LINKS_China for providing us with their parts:
                                        part BBa_K4011004 (TnaA-FL-FMO) and part BBa_K4011003 (Fre-SttH)</p><br>
                                    <img src="static/picture/engineering2.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <img src="static/picture/engineering3.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <p></p><br>
                                    <p>We used plasmids offered by LINKS_China to cut and ligation for obtaining plasmids expressing SttH and TnaA.</p>
                                    <br>
                                    <h3><a>Test</a></h3>
                                    <p>Firstly, we found that SttH alone had low reactivity because of mostly existing in an insoluble form[1]. Thus, we interspersed another ENGINEER process at the same time to solve this problem which is shown on the next part of engineer webpage.
                                        With literature survey and review, the test result shows that when 2.5 mM indole or Trp was used as substrate, 0.29 mM of 6-Br-indole or 0.78 mM of 6-Br-Trp was produced (Figure 2).</p><br>
                                    <img src="static/picture/engineering4.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 2 </a></h5>
                                        <span class="news-date">Concentrations of the substrates (indole or Trp) and the products (6-Br-indole or 6-Br-Trp) after the whole-cell reactions of ΔtnaA SttH+Fre using indole or Trp, respectively.</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <h3><a>Learn</a></h3>
                                    <p>Overall, the activity data suggest that the SttH reaction followed by the TnaA reaction (Trp→6-Br-Trp→6-Br-indole) give a higher yield of 6-Br-indole and would be advantageous for 6BrIG production over the other route (Trp→indole→6-Br-indole).</p><br>

                                </div><!-- .blog-post-news-description -->
                            </div><!-- .single-news-post-content -->
                        </div><!-- .single-news-post-wrapper -->
                    </article><!-- .saastrace-single-blog-post -->
                    <article class="saastrace-single-blog-post">
                        <div class="single-news-post-wrapper">
                            <div class="single-news-post-content">
                                <h2><a>Construction of Trp halogenase (SttH) and Fre fusion enzyme</a></h2>
                                <h3><a>Design</a></h3>
                                <div class="blog-post-news-description">
                                    <p>We found that SttH alone had low reactivity because of mostly existing in an insoluble form, possibly limiting the conversion of bromination up to 31.2% (0.78mM 6-Br-Trp, Figure 2)[1]. To improve the bromination reaction rate, we followed the designing of Lee et al to increase the solubility of SttH by constructing a fusion enzyme with a highly soluble protein at the N terminus of SttH (Figure 3). Fre was chosen as a fusion partner for SttH, since it is highly expressed in a soluble form in E. coli and can regenerate FADH2 efficiently.
                                    </p><br>
                                    <img src="static/picture/engineering5.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 3 </a></h5>
                                        <span class="news-date">SttH and Fre fusion enzyme with a flexible linker</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <h3><a>Build</a></h3>
                                    <p>Thanks to 2021 iGEM team LINKS_China for providing us with their parts:
                                        part BBa_K4011003 (Fre-SttH)
                                    </p><br>
                                    <img src="static/picture/engineering6.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <p></p><br>
                                    <h3><a>Test</a></h3>
                                    <p>Although the total expression of SttH decreased, the chimeric enzyme showed highly increased ratios of their soluble forms in E. coli with a higher field of 6-Br-trp. The molar concentrations of soluble Fre-L–SttH was 1.64 μM, which was more than three times higher than that of SttH by itself (0.53 μM)[1].
                                    </p><br>
                                    <h3><a>Learn</a></h3>
                                    <p>In summary, Fre was effective as a soluble N-terminal tag for SttH. And we choose Fre-SttH to replace alone SttH for Trp bromination in the last engineering process. </p><br>
                                    <img src="static/picture/engineering7.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 4 </a></h5>
                                        <span class="news-date">Reaction scheme of Trp bromination.
Trp 6-Hal requires oxygen, bromide salt and FADH2.
Flavin reductase (Fre) regenerates FAD to FADH2 using NAD(P)H.</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                </div><!-- .blog-post-news-description -->
                            </div><!-- .single-news-post-content -->
                        </div><!-- .single-news-post-wrapper -->
                    </article><!-- .saastrace-single-blog-post -->
                    <article class="saastrace-single-blog-post">
                        <div class="single-news-post-wrapper">
                            <div class="single-news-post-content">
                                <h2><a>Fermentation of Tyrian purple in two-cell system of E. coli </a></h2>
                                <h3><a>Design</a></h3>
                                <div class="blog-post-news-description">
                                    <p>The reported scheme of tyrian purple production in E.coli with the substrate of Trp is shown in Figure 5. We designed our optimization based on this scheme.
                                        As natural dye indigo and tyrian purple is hard to solute in water and cause difficulty in cloth dying. We applied the strategy that introduce a glucose moiety as a reversible protecting group to the reactive indigo and tyrian purple precursor indoxyl by glucosyltransferase PtUGT to form soluble indican as fermentation product (Figure 5). And β-glucosidase BGL is applied to remove the protecting group when cloth dying[2].
                                    </p><br>
                                    <img src="static/picture/engineering8.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 5 </a></h5>
                                        <span class="news-date">Reaction schemes of Trp 6-halogenase (Trp 6-Hal), tryptophanase (TnaA) and monooxygenase (MO)[1]</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <p>Also, for the steps using TnaA and MO (using one of special kinds named flavin-containing monooxygenase, MaFMO or FMO), we linked them with a flexible linker to form a chimeric enzyme for higher field and efficiency.
                                    </p><br>
                                    <img src="static/picture/engineering9.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 6 </a></h5>
                                        <span class="news-date">Optimized reaction scheme with the reversible protection of glucose</span>
                                    </div><!-- .single-news-post-meta --><br>
                                    <h3><a>Build</a></h3>
                                    <p>Thanks to 2021 iGEM team LINKS_China for providing us with their parts:
                                        part BBa_K4011004 (TnaA-FL-FMO) and part BBa_K4011003 (Fre-SttH)</p><br>
                                    <img src="static/picture/engineering10.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <img src="static/picture/engineering11.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <p></p><br>
                                    <p>We constructed plasmid pET28a-FMO-PtUGT to co-express PtUGT and FMO to convert indole or 6-Br-indole to indican or 6-Br-indican respectively. We also construct plasmid pET28a-BGL and pET28a-FMO to individually express BGL and FMO.</p>
                                    <br>
                                    <img src="static/picture/engineering12.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <img src="static/picture/engineering13.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <img src="static/picture/engineering14.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <br>
                                    <img src="static/picture/engineering15.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 7 </a></h5>
                                        <span class="news-date">Construction of pET28a-FMO-PtUGT</span>
                                    </div><!-- .single-news-post-meta --><br>
                                    <img src="static/picture/engineering16.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 8 </a></h5>
                                        <span class="news-date">Construction of pET28a-FMO</span>
                                    </div><!-- .single-news-post-meta --><br>
                                    <img src="static/picture/engineering17.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 9 </a></h5>
                                        <span class="news-date">Construction of pET28a-BGL</span>
                                    </div><!-- .single-news-post-meta --><br>
                                    <h3><a>Test</a></h3>
                                    <p>We transformed parts TnaA-FL-FMO and Fre-sttH into DH5α E.coli which endogenous TnaA gene have been knocked out. After culture and induced expression of these strains, we use the SDS-PAGE to separate and show the expression of TnaA-FL-FMO and Fre-sttH (Figure 10A). We found that we successfully express our fused proteins and these two proteins show high water solubility. </p><br>
                                    <p>We then culture strain that has been induced to express Fre-sttH by NPB solution and 2.5mM trp and 300mM NaBr as substrate. During fermentation of 24h, we use LC-MS to obtained data of concentration of trp and 6-Br-trp by every 6h (Figure 10B). Strain that has been induced to express TnaA-FL-FMO also be cultured by NPB solution and 1mM trp or 6-Br-trp as substrate. After fermentationfor 24h, we get obvious blue and purple dye in our culture tubes (Figure 10C). Besides, we attempt to produce tyrian purple by TnaA-FL-FMO using the culture after fermentation of trp and NaBr with Fre-sttH and the result has no difference from fermentation using standard 1mM 6-Br-trp (Figure 10D).</p><br>
                                    <p>Considering the bad solubility of indigo and tyrian purple, we apply 100% DMSO to solute the dye from cell precipitation and we get DMSO solution of indigo and tyrian purple (Figure 10E). Dye concentration is calculated by a standard curve measured by LINKS_China. We get a ~30% yield for indigo and ~20% yield for tyrian purple.
                                        We also use the tyrian purple we produce in cloth dying by put a piece of cloth into fermentation culture of Br-trp with TnaA-FL-FMO and we finally get the cloth in purple (Figure 10F). </p><br>
                                    <img src="static/picture/engineering18.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 10A </a></h5>
                                        <span class="news-date"> SDS-PAGE of TnaA-FL-FMO and Fre-sttH. WC stand for whole cell sample and S stand for supernatant sample after cell lysis</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <img src="static/picture/engineering19.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 10BCDEF </a></h5>
                                        <span class="news-date"> B) Concentration-time graph of trp and 6-Br-trp in culture of 2.5mM trp and 300mM NaBr with Fre-sttH. C) Tyrian purple (left) and indigo (right) produced by 6-Br-trp and trp with TnaA-FL-FMO. D) Tyrian purple produced by using culture of fermentation with Fre-sttH and trp. E) DMSO solution of indigo (top) and tyrian purple (bottom). F) Frabic dyed with tyrian purple.</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <h3><a>Learn</a></h3>
                                    <p>Our design of optimized reaction scheme is smooth and successful with a high yield of precursors indican, Br-indican and final product indigo and tyrian purple, as well as the excellent ability to dye fabrics evenly. Also, we can draw the conclusion that the constructed parts all work well and perform their correct function.
                                    </p><br>
                                </div><!-- .blog-post-news-description -->
                            </div><!-- .single-news-post-content -->
                        </div><!-- .single-news-post-wrapper -->
                    </article><!-- .saastrace-single-blog-post -->
                    <article class="saastrace-single-blog-post">
                        <div class="single-news-post-wrapper">
                            <div class="single-news-post-content">
                                <h2><a>Optimized fermentation of Tyrian purple in one-cell system of E. coli </a></h2>
                                <h3><a>Design</a></h3>
                                <div class="blog-post-news-description">
                                    <p>Due to TnaA can react with both tryptophan and the first step product bromotryptophan, Fre-SttH and TnaA has to be in two different strains. So the two-cell system, ΔTnaA E. coli strain without endogenous TnaA expression and another ΔTnaA E. coli strain with extra TnaA expression, are used to produce tyrian purple with less byproduct indigo. That’s why Lee et al have to use two-cell system to complete the production of tyrian purple.
                                    </p><br>
                                    <p>Considering the problem of tryptophanase TnaA lack of specificity, we want to use muted trp repressor that can only bind trp to sense the concentration of trp and 6-Br-trp in E. coli to control the expression of TnaA. We optimized two-cell sysytem to one-cell system with a spatio-temporal well-controlled TnaA synthesis (Figure 11). We applied a synthetic phylogeny of programmable Trp repressor with directed evolution to E.coli, which simplified the process of reacting with bromotryptophan not tryptophan[3].</p><br>
                                    <p>Compartmentalized partnered replication (CPR) is an emulsion-based directed evolution method based on a robust and modular phenotype–genotype linkage. In contrast to other in vivo directed evolution approaches, CPR largely mitigates host fitness effects due to a relatively short expression time of the gene of interest. CPR is based on gene circuits in which the selection of a 'partner' function from a library leads to the production of a thermostable polymerase. After library preparation, bacteria produce partner proteins that can potentially lead to enhancement of transcription, translation, gene regulation, and other aspects of cellular metabolism that reinforce thermostable polymerase production. Individual cells are then trapped in water-in-oil emulsion droplets in the presence of primers and dNTPs, followed by the recovery of the partner genes via emulsion PCR. In this step, droplets with cells expressing partner proteins that promote polymerase production will produce higher copy numbers of the improved partner gene. The resulting partner genes can subsequently be recloned for the next round of selection[4].</p><br>
                                    <p>Iterative rounds of positive and negative compartmentalized partnered replication (CPR) led to the exponential amplification of variants that responded with high affinity and specificity to halogenated tryptophan analogs and novel operator sites. We empower cells to perceive the relative ratio of bromotryptophan to tryptophan to control TnaA synthesis. </p><br>

                                    <img src="static/picture/engineering20.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 11 </a></h5>
                                        <span class="news-date">Optimized reaaction scheme of one-cell system adding directed evolution </span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <h3><a>Build</a></h3>
                                    <p>We firstly constructed three plasmids pTrpR, pPOS and pNEG. pTrpR is induced to express muted trp repressors. Active trp repressors with its ligand can induce pPOS to express Taq polymerase and inactive trp repressors or trp repressor without its ligand can induce pNEG to express Taq polymerase (Figure 12).
                                    </p><br>
                                    <p>For GFP test of CPR cycle, we constructed pNEG-sfGFP and pPOS-sfGFP which replace Taq polymerase gene with sfGFP to characterize the expression of Taq polymerase.</p><br>
                                    <p>For the mock selection of CPR cycle, we inserted a DNA sequence of 200bp into wild type trp repressor to construct INAC-pTrpR (Figure 18). We mix plasmid pTrpR with wild type trp repressor and INAC-pTrpR with a mass ratio of 1:1 and transform the mixture into pNEG and pPOS competent cells. Induced cells were resuspend by CPR mix and emulsion droplets were constructed by mixing CPR mix and oil mix and vortex for 4 min. </p><br>
                                    <img src="static/picture/engineering21.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 12 </a></h5>
                                        <span class="news-date">Scheme of CPR positive and negative circuits </span>
                                    </div><!-- .single-news-post-meta -->
                                    <p></p><br>
                                    <img src="static/picture/engineering22.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 13 </a></h5>
                                        <span class="news-date">Construction of pTrpR</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <img src="static/picture/engineering23.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 14 </a></h5>
                                        <span class="news-date">Construction of pPOS</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <img src="static/picture/engineering24.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 15 </a></h5>
                                        <span class="news-date"> Construction of pNEG</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <img src="static/picture/engineering25.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 16 </a></h5>
                                        <span class="news-date">  Construction of pPOS-GFP</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <img src="static/picture/engineering26.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 17 </a></h5>
                                        <span class="news-date">Construction of pNEG-GFP</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <img src="static/picture/engineering27.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 18 </a></h5>
                                        <span class="news-date">Scheme of INAC-pTrpR construction</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <h3><a>Test</a></h3>
                                    <p>(1) GFP test
                                    </p><br>
                                    <p>The expression of Taq polymerase can not be monitored easily, we apply GFP assay to use GFP production to analyzing the gene circuits function. We transform pTrpR which express wild type trp repressor into competent cells that have been transformed with plasmid pNEG-sfGFP or pPOS-sfGFP. Then we culture and induce with different concentration of trp and 1mM IPTG to induce expression of sfGFP. We expect that as trp concentration increases, the expression of sfGFP (RPU/OD600) in pNEG-sfGFP gradually decreases, and the expression of sfGFP (RPU/OD600) in pPOS-sfGFP gradually increases (Figure 19A).
                                    </p><br>
                                    <p>(2) Mock selection</p><br>
                                    <p>In addition to GFP test, we also perform a mock selection to confirm the designed gene circuit function can actually select positive clone. At first, we mix culture with strains that has been induced to express sfGFP and oil mix to check the quality of emulsion droplet. We measure the diameter of droplet with a phase contrast microscopy. We found that the diameter of emulsion droplets are range about 2-10 μm and less than 20% emulsion droplets contain a GFP-expression cell (Figure 19B). Thus, we have checked any emulsion droplet can only contain a single cell. We insert a random sequence into wild type trp repressor to make it inactive to bind to trp, and we get plasmid INAC-pTrpR. We mix plasmid pTrpR with wild type trp repressor and INAC-pTrpR with a mass ratio of 1:1 and transform the mixture into pNEG and pPOS competent cells. A mock CPR seletion is next carried out. Induced strains is emulsified with emulsion mix and replicate positive clone by ePCR. The emulsion remains in a phase-independent state during ePCR (Figure 19C). After breaking the emulsion by adding ddH2O and chloroform, we apply a recovery PCR and perform a agarose electrophoresis to monitor the result of the mock selection. PNEG give an obvious amplification of inactive trp repressor and pPOS give an obvious amplification of active trp repressor (Figure 19D).
                                    </p><br>
                                    <img src="static/picture/engineering28.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 19A </a></h5>
                                        <span class="news-date">GFP production (RPU/OD600) – trp concentration graph by GFP test for wild type trp repressor and pNEG (due to time and pandemic, we only completed the experiment of pNEG). </span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <img src="static/picture/engineering29.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 19B </a></h5>
                                        <span class="news-date"> Phase contrast microscopy image of emulsion droplet of 50×diluted scale bar, 10μm (left) of emusion droplet merge with GFP expression of 1× diluted emulsion scar bar, 10μm.  </span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <img src="static/picture/engineering30.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 19C </a></h5>
                                        <span class="news-date">From left to right: cell culture emulsified with emulsion mix; Cell emulsion after ePCR circle; Emulsion after perform centrifuge; Phase-separate solution after breaking emulsion. </span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <img src="static/picture/engineering31.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 19D </a></h5>
                                        <span class="news-date"> result of agarose electrophoresis between trpR:INAC-trpR = 1:1 and trpR and inactive trpR after mock selection with pNEG (due to time and pandemic, we only completed the experiment of pNEG) </span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <h3><a>Learn</a></h3>
                                    <p>We found that high concentration of trp can inhibit expression of sfGFP in negative selection system. However, it can only provide about three times the inhibition effect which cannot achevie screen effect. Possibly, trp repressor has a low expression or trp repressor with trp cannot strongly bind to trp operator at DNA. Same experiments of positive selection system haven't be carry out. </p><br>
                                    <p>Also, we have successfully make emulsion droplets of high quality. However, results of mock selection is not obvious which means selection system cannot distinguish wild type trp repressor with inactive trp repressor that consistent with results of GFP test.
                                    </p><br>
                                </div><!-- .blog-post-news-description -->
                            </div><!-- .single-news-post-content -->
                        </div><!-- .single-news-post-wrapper -->
                    </article><!-- .saastrace-single-blog-post -->
                    <article class="saastrace-single-blog-post">
                        <div class="single-news-post-wrapper">
                            <div class="single-news-post-content">
                                <h2><a>Directed evolution of trp repressor mutation library </a></h2>
                                <h3><a>Design</a></h3>
                                <div class="blog-post-news-description">
                                    <p>After testing the CPR circuit works well, we are going to introduce directed evolution to trp repressor. Our libraries are designed in which residues (I57, V58, R54, L60, G85, T81, I82) that might contact trp at its 6C positions of the indole ring due to their proximity. Considering several position overlapped with those involved in the formation of the HTH motif, we initially choose I57, V58 and T81 as our mutant site for directed evolution. We perform site-directed saturation mutation on these three residues by completely randomized through re-synthesis of the gene with NNS codons (N = A, T, G, C; S = G, C). The library (32768 variants) are transformed into competent cells that have been transformed with pNEG or pPOS beforehand and carry out several circle of CPR to select positive clone we want (Figure 20). Due to the restriction of time, we have not perform a selective circle in our project, we plan to simulate this process using mathematical modeling.
                                    </p><br>
                                    <img src="static/picture/engineering8.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 5 </a></h5>
                                        <span class="news-date">Reaction schemes of Trp 6-halogenase (Trp 6-Hal), tryptophanase (TnaA) and monooxygenase (MO)[1]</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <p>Also, for the steps using TnaA and MO (using one of special kinds named flavin-containing monooxygenase, MaFMO or FMO), we linked them with a flexible linker to form a chimeric enzyme for higher field and efficiency.
                                    </p><br>
                                    <img src="static/picture/engineering32.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 20 </a></h5>
                                        <span class="news-date">3D view of trpR binding to DNA and the trp binding site</span>
                                    </div><!-- .single-news-post-meta --><br>
                                    <h3><a>Build</a></h3>
                                    <p>More details in Drylab.</p><br>
                                    <h3><a>Test</a></h3>
                                    <p>At first, we modeling the process of Transcription and translation process. We used the ODE solver “deSolve” in R to simulate our positive selection system based off the 9 ODEs we wrote and “ggplot2” to plot the dynamics of molecules of interest, and obtained the results below: As shown by the graph (Figure 21A), within 2 hours, TrpR reaches its plateau at about 150 uM and at the same time TrpR gets rapid polymerization formation into TrpR2, which peaks at Time = 20h, 75 uM. Compared with TrpR and TrpR2, TrpR2-T's formation process is much lower, reaching it saturated at Time = 40h, 25 uM. In the CI, CI2 graph (Figure 21B), CI quickly reaches its peak at Time = 8h, 3.8 uM and then it go through a steady but slow decreasing process to about 1.7 uM. While for the CI2 protein, it has the similar trend with CI and its peak and steady state concentration is half of that of CI. While for the GFP graph (Figure 21C), we could see in the first day (within 24h), GFP quickly peaks at about 2 uM, while after that it experiences a significant drop to no more than 0.5 uM. Soon, GFP quickly get translation.</p><br>
                                    <p>Then, we model the influence of Trp's concentration in directed evolution system X aix means Trp increase from 10-4 to 1, Y aix means relatively expression level (Figure 21D). In the left figure, we can see, when Trp's concentration is under 20 × 10-3 uM. CI2 protein is at its saturate state. When Trp increases to 30 × 10-1 uM, CI2 experiences a significant drop, from 2.0 uM to 0.5 uM. In the right figure, GFP's relative expression process is contrary to CI2 trend. Below Trp concentration 20 × 10-3 uM, GFP's expression is close to 0. But as trp's concentration increases. </p><br>
                                    <img src="static/picture/engineering33.png" alt="Blog One" style="width: 50%;margin-left: 25%">
                                    <div class="single-news-post-meta">
                                        <h5 class="news-tag"><a href="#">FIGURE 21 </a></h5>
                                        <span class="news-date"> A) TrpR, TrpR2 and TrpR2-T expression changed with time   B) CI and CI2 expression changed with time  C) GFP expression changed with time D) CI2 and GFP expression changed with increased Trp concentration.</span>
                                    </div><!-- .single-news-post-meta -->
                                    <br>
                                    <h3><a>Learn</a></h3>
                                    <p>With the help of modeling, we know better about this directional evolution and selection system. As seen in the results, our modeling shows how the positive screening system works. TrpR and TrpR2 quickly reach its saturate state with proper Trp concentration, and TrpR2 and Trp's polymer can generate, which can inhibit the expression of CI2. Also, Taq enzyme can be expressed. With increasing Trp concentration, the relative expression of GFP has a similar trend with experiment data in wetlab. According to this, we could estimate the combined rate between TrpR2 and Trp, which has a huge benefit to our gene design.
                                    </p><br>
                                </div><!-- .blog-post-news-description -->
                            </div><!-- .single-news-post-content -->
                        </div><!-- .single-news-post-wrapper -->
                    </article><!-- .saastrace-single-blog-post -->
                    <article class="saastrace-single-blog-post">
                        <div class="single-news-post-wrapper">
                            <div class="single-news-post-content">
                                <h2><a href="blog-single.html">Summary</a></h2>
                                <div class="blog-post-news-description">
                                    <p>Our project optimized the synthesis biology approach to product tyrian purple and verified feasibility of synthesizing tyrian purple with a one-cell system. Mathematical modeling results can be used as a reference of quantitative data in wetlab.
                                    </p><br>
                                </div><!-- .blog-post-news-description -->
                            </div><!-- .single-news-post-content -->
                        </div><!-- .single-news-post-wrapper -->
                    </article><!-- .saastrace-single-blog-post -->
                    <article class="saastrace-single-blog-post">
                        <div class="single-news-post-wrapper">
                            <div class="single-news-post-content">
                                <h2><a href="blog-single.html">References</a></h2>
                                <div class="blog-post-news-description">
                                    <p>[1] Lee, J., Kim, J., Song, J. E., Song, W. S., Kim, E. J., Kim, Y. G., Jeong, H. J., Kim, H. R., Choi, K. Y.,    & Kim, B. G. Production of Tyrian purple indigoid dye from tryptophan in Escherichia coli. Nature chemical biology, 2021, 17(1), 104-112.
                                        DOI: https://doi.org/10.1038/s41589-020-00684-4<br>
                                        [2] Hsu, T. M., Welner, D. H., Russ, Z. N., Cervantes, B., Prathuri, R. L., Adams, P. D., & Dueber, J. E. Employing a biochemical protecting group for a sustainable indigo dyeing strategy. Nature chemical biology, 2018, 14(3), 256–261.
                                        DOI: https://doi.org/10.1038/nchembio.2552<br>
                                        [3] Ellefson, J. W., Ledbetter, M. P., & Ellington, A. D. Directed evolution of a synthetic phylogeny of programmable Trp repressors. Nature chemical biology, 2018, 14(4), 361–367.
                                        DOI: https://doi.org/10.1038/s41589-018-0006-7<br>
                                        [4] Abil, Z., Ellefson, J. W., Gollihar, J. D., Watkins, E., & Ellington, A. D. Compartmentalized partnered replication for the directed evolution of genetic parts and circuits. Nature protocols, 2017, 12(12), 2493–2512.
                                        DOI: https://doi.org/10.1038/nprot.2017.119</p><br>
                                </div><!-- .blog-post-news-description -->
                            </div><!-- .single-news-post-content -->
                        </div><!-- .single-news-post-wrapper -->
                    </article><!-- .saastrace-single-blog-post -->
                </div><!-- .saastrace-blog -->
            </div><!-- .col-lg-9 -->
            <div class="col-lg-3">
                <aside class="saastrace-blog-sidebar">
                    <div class="widget recent_posts">
                        <h4>Engineering</h4>
                        <ul>
                            <li><a href="#">Overview</a></li>
                            <li><a href="#">Establishing a pathway from Trp to 6-bromoindole</a></li>
                            <li><a href="#">Construction of Trp halogenase (SttH) and Fre fusion enzyme</a></li>
                            <li><a href="#">Fermentation of Tyrian purple in two-cell system of E. coli </a></li>
                            <li><a href="#">Optimized fermentation of Tyrian purple in one-cell system of E. coli </a></li>
                            <li><a href="#">Directed evolution of trp repressor mutation library</a></li>
                            <li><a href="#">Summary</a></li>
                            <li><a href="#">References</a></li>
                        </ul>
                    </div><!-- .recent-posts -->
                </aside><!-- .saastrace-blog-sidebar -->
            </div><!-- .col-lg-3 -->
        </div><!-- .row -->
    </div><!-- .container -->
</div><!-- .saastrace-blog-area -->


<!-- ================================================================= -->
<!-- ====================== Saastrace Footer Area ==================== -->
<!-- ================================================================= -->


<footer class="saastrace-footer-area">
    <img src="static/picture/footerpic.png" alt="Laptop" style="width: 100%">
    <div class="col-md">
        <div class="footer-left wow fadeInRight" data-wow-delay="100ms">
            <p style="color: #0a0319; padding: 5px">Copyright &copy; 2022 SUSTech_Shenzhen Design</p>
        </div><!-- .footer-left -->
    </div><!-- .col -->

</footer><!-- .saastrace-footer-area -->

<script src="static/js/modernizr-3.7.1.min.js"></script>
<script src="static/js/jquery-3.3.1.min.js"></script>
<script src="static/js/bootstrap.min.js"></script>
<script src="static/js/swiper.min.js"></script>
<script src="static/js/fontawesome.min.js"></script>
<script src="static/js/classie.js"></script>
<script src="static/js/saastrace-search.js"></script>
<script src="static/js/slinky.min.js"></script>
<script src="static/js/wow.min.js"></script>
<script src="static/js/app.js"></script>
</body>

</html>